go.mod

```diff
@@ -0,0 +1,79 @@
+module common-instancetypes/tests
+
+go 1.20
+
+require (
+	github.com/onsi/ginkgo/v2 v2.13.0
+	github.com/onsi/gomega v1.29.0
+	k8s.io/api v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
`v12.0.0+incompatible` is a hack required in older projects, we're starting fresh. Just point it to the latest version.